ironfish-cli/src/commands/accounts/rescan.ts

```diff
@@ -22,34 +22,56 @@ export class RescanCommand extends IronfishCommand {
       description:
         'clear the in-memory and disk caches before rescanning. note that this removes all pending transactions',
     }),
+    offline: flags.boolean({
```

Comment:
This should probably be named `local` to match up with the similar flag in the `config` commands (e.g. `config:get`)